pkg/serviceaccount/claims.go

```diff
@@ -23,11 +23,20 @@ import (
 
 	"gopkg.in/square/go-jose.v2/jwt"
 	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/serviceaccount/metrics"
 
 	apiserverserviceaccount "k8s.io/apiserver/pkg/authentication/serviceaccount"
 	"k8s.io/kubernetes/pkg/apis/core"
 )
 
+const (
+	// Special injected bound service account token expiration to trigger safe rollout of its time-bound feature.
+	SpecialBoundTokenExpirationSeconds = 60*60 + 7
```

Comment:
Why adding 7 for this value ?

Reply:
To make it prime, thus harder for someone to randomly stumble on.